src/masonite/routes/Route.py

```diff
@@ -24,7 +24,7 @@ def get(self, url, controller, module_location=None, **options):
         return HTTPRoute(
             url,
             controller,
-            request_method=["get"],
+            request_method=["get", "head"],
```

Comment:
Not sure i like doing this. This feature seems to tightly couples to other features

Reply:
Above item and this one should be discussed in the PR https://github.com/MasoniteFramework/masonite/pull/578.

The HEAD thing is a prerequisite to this PR and is implemented in #578.